docs/content/Deployment/Platform-Docker.md

```diff
@@ -0,0 +1,328 @@
+---
+title: Deploy with Docker
+permalink: /deployment/platforms/docker
+category: Deployment
+subCategory: Platforms
+menuOrder: 1
+---
+
+This guide walks you through deploying Cube.js with Docker.
+
+## Prerequisites
+
+- Docker CLI
+
+## Configuration
+
+Create a Docker Compose stack by creating a `docker-compose.yml`. An example
+stack using BigQuery as a data source is provided below:
```

Comment:
The example config is quite lengthy and contains a lot of things, so it might make sense to explain what's going to be run: a Cube.js API instance, a refresh worker, etc.